internal/services/web/app_service.go

```diff
@@ -433,6 +433,12 @@ func schemaAppServiceSiteConfig() *pluginsdk.Schema {
 					Optional: true,
 				},
 
+				"vnet_route_all_enabled": {
+					Type:     pluginsdk.TypeBool,
+					Optional: true,
+					Computed: true,
+				},
```

Comment:
does this want to be computed? feels like since Azure's forcing people over this should be set in configs too?

Reply:
Ideally, no. But this method lets users that currently set via `app_settings` have their configs still work and update when they're ready/able?